<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>關鍵字查詢</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QComboBox" name="comboBox">
              <property name="editable">
               <bool>true</bool>
              </property>
              <item>
               <property name="text">
                <string>中文</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>英文</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_title"/>
            </item>
            <item>
             <widget class="QPushButton" name="p_But_by_title">
              <property name="text">
               <string>查詢</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,1">
         <item>
          <widget class="QTableView" name="tableView">
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>300</number>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>75</number>
           </attribute>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,3,1">
           <item>
            <widget class="QPushButton" name="p_But_firstpage">
             <property name="text">
              <string>第一頁</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="p_But_uppage">
             <property name="text">
              <string>上一頁</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="p_But_downpage">
             <property name="text">
              <string>下一頁</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="p_But_lastpage">
             <property name="text">
              <string>最後一頁</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_page"/>
           </item>
           <item>
            <widget class="QLabel" name="page">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>頁數：</string>
             </property>
             <property name="textFormat">
              <enum>Qt::RichText</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="total">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>總筆數：</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionEXIT"/>
    <addaction name="actionSave_Data"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEXIT">
   <property name="text">
    <string>EXIT</string>
   </property>
  </action>
  <action name="actionSave_Data">
   <property name="text">
    <string>Save Data</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
